rewardbench/rewardbench.py

```diff
@@ -389,8 +414,9 @@ def main():
 
         with open(output_path, "w") as f:
             for chosen, rejected in zip(scores_chosen, scores_rejected):
-                f.write(json.dumps({"chosen": scores_chosen, "rejected": scores_rejected}) + "\n")
+                f.write(json.dumps({"chosen": chosen, "rejected": rejected}) + "\n")
```

Comment:
Was this just a bug?

Reply:
Yeah I think so, 

Previous: 

<img width="1400" alt="image" src="https://github.com/user-attachments/assets/3711bbf8-a6fc-4c13-aadd-0f6dd2edb3a8">

After the change in this PR:

<img width="735" alt="image" src="https://github.com/user-attachments/assets/a0ecb15e-bc13-4ca2-a04e-39a1754d76dc">